### ISSUE TITLE
Everything seeeeeems to be working fine, but funding stays $0.00

### ISSUE THREAD

**Issue by NeedMyMoney** [ITPrism/CrowdFunding#147]
Hello,

I really love the crowdfunding script. I think even the free version is AMAZING... until I ran into this problem. I have started receiving donations. I receive the money from PayPal. PayPal is the only plugin that I am using, but I do not show the transactions added and the funding number remains $0.00. Please help. How do I solve this problem? My website is: http://www.AskSharifah.com

I also need to know if I can manually add the donations that I have already received in.

Thanks

**Comment by NeedMyMoney** [comment]
I believe this is the error but I don't know what it means or how to fix it:

#
#<?php die('Forbidden.'); ?>
#Date: 2015-05-19 11:22:29 UTC
#Software: Joomla Platform 13.1.0 Stable [ Curiosity ] 24-Apr-2013 00:00 GMT

#Fields: datetime	priority	category	message
2015-05-19T11:22:29+00:00	INFO	controller	Holding edit ID com_plugins.edit.plugin.10022 Array (     [0] => 10022 ) 
2015-05-19T11:22:29+00:00	INFO	controller	Checking edit ID com_plugins.edit.plugin.10022: 1 Array (     [0] => 10022 ) 
2015-05-19T11:22:48+00:00	INFO	controller	Releasing edit ID com_plugins.edit.plugin.10022 Array ( )

**Comment by NeedMyMoney** [comment]
I debugged it and this is what I received Invalid transaction request method (GET). PAYPAL_PAYMENT_PLUGIN_DEBUG

**Comment by ITPrism** [comment]
Hi,
Thank you for the post!

You gets "Invalid transaction request method" when you try to load the notification script from your browser.

The notification script should be visible from PayPal servers. Do you have for any restrictions by IP or htaccess passwords?

If you would like, you can create [Sandbox account] (https://developer.paypal.com/webapps/developer/applications/accounts) and make test transactions. Enable debug mode of your Joomla. That will give more information about the problem.

You can also check IPN history.
https://developer.paypal.com/docs/classic/ipn/integration-guide/IPNOperations/

Best regards,
Todor

**Comment by NeedMyMoney** [comment]
I do not believe that I have any IP or htaccess password restrictions. Everything else works perfectly How would I confirm?

When I go to this page it's just blank: index.php?option=com_crowdfunding&task=notifier.notify&format=raw&payment_service=PayPal 

The Sandbox and IPN is too over my head. How do I fix this problem?

**Comment by ITPrism** [comment]
Hi,
Create PayPal sandbox account.
https://developer.paypal.com/docs/classic/lifecycle/sb_create-accounts/

Enable Sandbox on your paypal plugin.

Debug the system.
http://itprism.com/help/95-crowdfunding-documentation-faq#debug

You will find information about problem in logs section.
Alternatively, you can find information about problem in IPN History.
http://itprism.com/help/95-crowdfunding-documentation-faq#debug

Just check for error messages. We need to know more about the error.

**Comment by ITPrism** [comment]
Best practices is to test with sandbox before start your website.

**Comment by NeedMyMoney** [comment]
I have the error. I keep getting the same error. Invalid transaction request method (GET).
I deleted my .htacess file. What else can I do? What else would the error tell you?

#
#<?php die('Forbidden.'); ?>
#Date: 2015-05-19 11:22:29 UTC
#Software: Joomla Platform 13.1.0 Stable [ Curiosity ] 24-Apr-2013 00:00 GMT

#Fields: datetime	priority	category	message
2015-05-19T11:22:29+00:00	INFO	controller	Holding edit ID com_plugins.edit.plugin.10022 Array (     [0] => 10022 ) 
2015-05-19T11:22:29+00:00	INFO	controller	Checking edit ID com_plugins.edit.plugin.10022: 1 Array (     [0] => 10022 ) 
2015-05-19T11:22:48+00:00	INFO	controller	Releasing edit ID com_plugins.edit.plugin.10022 Array ( )

**Comment by NeedMyMoney** [comment]
Everything works perfectly. I even receive the money, but for some reason PayPal is not notifying me. I saw that several people had this issue. Yet, I never saw their issue fixed. I just saw the issues closed. Please HELP!

**Comment by NeedMyMoney** [comment]
I fixed the problem... YAAAAAAY

I actually had to update my PayPal IPN settings. Once I added the Notify url to PayPal it worked. Then I had PayPal resend the transactions and it WORKED!!  The amounts were even updated on Funding... Thank you!! Thank you!! Thank you!!!

**Comment by ITPrism** [comment]
Hmm... The Notify URL is provided by plugin form settings dynamically. 
It is not necessary to setup Notify URL on your PayPal account.
It may be depends from account or there are some changes. I have to check that.

Which payment plugin do you use - standard, express or adaptive? What PayPal account do you use - business, personal,...?

**Comment by NeedMyMoney** [comment]
OH YES! I am so excited.. So, so excited. Now everything is working.

I have PayPal business. I actually had to go into My Profile and select the IPM settings and turn them on and I had to add the return URL to my account. I hadn't done any of that before. I thought that once I added the information into your system it would just work. But it's fixed now! YES!!

I want to upgrade my account. What is your charge for installation?

**Comment by NeedMyMoney** [comment]
I am using PayPal Standard.

**Comment by ITPrism** [comment]
Yeah, it should work once you add an information into the plugin settings. I am not sure why it does not work in your case.

I am going to do a research.

Thank you very much!

**Comment by 52ponds** [comment]
Hi -

I'm having this same issue.

I checked with the server IT guys who said that what Paypal is generating is getting through to the server: http://mysite.com/index.php?option=com_crowdfunding&task=notifier.not
ify&format=raw&payment_service=PayPal

I turned on the Joomla debug function on the site, but it's not showing any errors.

I've tested this with a live account at Paypal, paying $1 each time. The payment goes through fine and then returns back to the site (I'm using Paypal Standard), but the amount in the project total isn't being updated.

I'm using 1.11.1 Pro ( Stable )

Thoughts?

Thanks!
